test/asyncio_tests/test_examples.py

```diff
@@ -902,16 +902,11 @@ async def run_transaction_with_retry(txn_coro, session):
                     await txn_coro(session)  # performs transaction
                     break
                 except (ConnectionFailure, OperationFailure) as exc:
-                    print("Transaction aborted. Caught exception during "
-                          "transaction.")
-
-                    # If transient error, retry the whole transaction
-                    if exc.has_error_label("TransientTransactionError"):
-                        print("TransientTransactionError, retrying"
-                              "transaction ...")
-                        continue
-                    else:
+                    print('Transaction aborted. Caught exception during transaction.')
+                    if not exc.has_error_label('TransientTransactionError'):
                         raise
+                    print('TransientTransactionError, retryingtransaction ...')
+                    continue
```

Comment:
For this file I prefer the old code since it more easily fits in 80 chars. What was the pep issue here? Could you revert it?

Could you also fix the missing space in the "retryingtransaction" error message?

Reply:
I don't use any tool, just some Manual refactoring (old school methods docs and some references), i will fix this and then you can re-review it

Reply:
@ShaneHarvey fixed